eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/ResourceCollectionClientProvider.cs

```diff
@@ -262,10 +262,10 @@ private MethodBodyStatement BuildReturnStatementsForGetIfExists(ValueExpression
             return statements;
         }
 
-        private MethodBodyStatement BuildReturnStatementsForExists(ValueExpression responseVariable)
+        private IReadOnlyList<MethodBodyStatement> BuildReturnStatementsForExists(ValueExpression responseVariable)
         {
             var returnValueExpression = responseVariable.Property("Value").NotEqual(Null);
-            return Return(Static(typeof(Response)).Invoke(nameof(Response.FromValue), returnValueExpression, responseVariable.Invoke("GetRawResponse")));
+            return [Return(Static(typeof(Response)).Invoke(nameof(Response.FromValue), returnValueExpression, responseVariable.Invoke("GetRawResponse")))];
```

Comment:
nit: could we spread the elements into multiple lines so that the statements could be more clear?